### PR TITLE
[SDS-307] Now able to get calibration data from result

### PR DESCRIPTION
**Pull request by QFer**
* Added method to get calibration info to api
* Added unit tests

### CODE REVIEW

File: src/quantuminspire/api.py [hunk 1]
```diff
@@ -676,6 +676,32 @@ def get_measurement_register_from_result(self, result_id: int) -> List[Any]:
             raise ApiError(f'Measurement register for result with id {result_id} does not exist!') from err_msg
         return measurement_register
 
+    def get_calibration_from_result(self, result_id: int) -> Optional[Any]:
+        """ Gets the calibration information of the executed cQASM code, given the result_id.
```
Comment (peendebak) [inline src/quantuminspire/api.py:680]:
```suggestion
        """ Gets the calibration information of the executed code, given the result_id.
```
We also submit qiskit code (right that that is converted to cqasm internally, but that might change)

File: src/quantuminspire/api.py [hunk 2]
```diff
@@ -676,6 +676,32 @@ def get_measurement_register_from_result(self, result_id: int) -> List[Any]:
...
+
+        :return:
+            The calibration information from the backend.
+            None is returned when there is no calibration information (e.g. simulator backend).
+        """
+        calibration_info = None
+        result = self.get_result(result_id)
+        calibration_url = result.get('calibration')
+        if calibration_url is not None:
+            try:
```
Comment (peendebak) [inline src/quantuminspire/api.py:695]: I guess this code works, but is the parsing of the url needed? One can just do
```
result = requests.get(c, auth=authentication)
calibration = result.json()
```
(the `authentication` and `.json` can be wrapped in something off course)

Comment (QFer) [reply src/quantuminspire/api.py:695]: Yes, you can directly use request package to send a get for this url,, but I think it is better to use the already authenticated coreapi consistently for all communication with the backend api as it is now done in SDK.

File: src/quantuminspire/api.py (hunk 1 above)
Comment (QFer) [reply src/quantuminspire/api.py:680]: Changed it to executed job. Also changed some other instances of cQASM (where it is not needed to be called cQASM explicitly)

**Review by peendebak — DISMISSED** [review]
Functionality here is good. There is still the issue of documenting the (possible) confusion between the job_id and project_id, but that should be done in another PR.

**Comment by QFer** [comment]
Had to force push after rebase

**Review by oschusler — APPROVED** [review]

**Pull request merged**